### PR TITLE
Refs #603 ~ User controller implemented.

### PR DESCRIPTION
**Pull request by Netmisa**
# Introduction
*User controller implemented*
*Model cleaned.*

```
POST   /users
GET    /users
GET    /users/1
PATCH  /users
DELETE /users/1
```
 
## 1. Installation
*Execute command:*
```shell
bin/console doctrine:database:drop --env test --force
bin/console doctrine:database:create --env test
bin/console doctrine:schema:update --env test --force
```

## 2. Run functional/unit tests
*Execute command:*
```shell
vendor/bin/phpunit -c phpunit.xml.dist
```

### CODE REVIEW

**Comment by Netmisa** [comment]
Jenkins test this please

**Comment by lrocheWB** [comment]
@Netmisa can you push "bin/phpunit" file ? Not found in your branch... I can't run unit tests :'(

**Comment by Netmisa** [comment]
Jenkins test this please

File: src/ApiBundle/Controller/UsersController.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use FOS\RestBundle\Controller\FOSRestController;
+use FOS\RestBundle\Request\ParamFetcherInterface;
+use FOS\RestBundle\Controller\Annotations\RequestParam;
+use FOS\RestBundle\Controller\Annotations\QueryParam;
+use ApiBundle\Entity\User;
+
+class UsersController extends FOSRestController
+{
+    public function getUsersAction(ParamFetcherInterface $paramFetcher)
```
Comment (lrocheWB) [inline src/ApiBundle/Controller/UsersController.php]: $paramFetcher useless...

**Comment by Netmisa** [comment]
Jenkins could you test this please ?

**Comment by Netmisa** [comment]
Jenkins could you test this please ?

**Pull request merged**